public/components/trace_analytics/components/services/service_view.tsx

```diff
@@ -76,6 +77,18 @@ export function ServiceView(props: ServiceViewProps) {
   const [redirect, setRedirect] = useState(false);
   const [actionsMenuPopover, setActionsMenuPopover] = useState(false);
 
+  let serviceId: string | null = null;
+
+  try {
+    const location = useLocation();
```

Comment:
> Location is causing the service view page to not render as it is not defined, so it needs to be within the "Try".

do you have more details on what the error is? is it happening when calling `useLocation`?

Reply:
Resolved with moving the flyout component into router.